site/_utils/tag-11tydata.js

```diff
@@ -41,13 +41,10 @@ module.exports = locale => ({
       let paginated = [];
 
       for (const tag of tags) {
-        const posts = tag.posts[locale];
-        if (posts.length > 0) {
-          paginated = paginated.concat(
-            addPagination(posts, locale + '/tags/' + tag.key, {
-              title: tag.isGeneratedTag ? tag.title : i18n(tag.title, locale),
-            })
-          );
+        tag['elements'] = tag.posts[locale];
```

Comment:
why quote?

Reply:
Elements doesn't exist on a Tag, but I need it to for the pagination function. This is a temp solution until I uses `postsData` for tags as well.